src/components/editor/Bindings/AdvancedOptions/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Stack, Typography } from '@mui/material';
+import WrapperWithHeader from 'components/shared/Entity/WrapperWithHeader';
+import ErrorBoundryWrapper from 'components/shared/ErrorBoundryWrapper';
+import { useEntityType } from 'context/EntityContext';
+import { useIntl } from 'react-intl';
+import { useBindingStore } from 'stores/Binding/Store';
+import OnIncompatibleSchemaChange from '../OnIncompatibleSchemaChange';
+import TimeTravel from '../TimeTravel';
+import { AdvancedOptionsProps } from './types';
+
+export default function AdvancedOptions({
+    bindingIndex,
+    bindingUUID,
+    collectionName,
+}: AdvancedOptionsProps) {
+    const intl = useIntl();
+
+    const entityType = useEntityType();
+
+    const fullSourceErrorExists = useBindingStore((state) => {
+        const fullSourceErrors = state.fullSourceConfigs[bindingUUID]?.errors;
+
+        if (!fullSourceErrors) {
+            return false;
+        }
+
+        return fullSourceErrors.length > 0;
+    });
+
+    const onIncompatibleSchemaChangeErrorExists = useBindingStore(
+        (state) => state.onIncompatibleSchemaChangeErrorExists.binding
+    );
```

Comment:
Do not need to change for this PR... but I think it would be safe to put these two into a single `shallow` call. I think that might help reduce how often these might change.